changelog/wave/v1.16.2.mdx

```diff
@@ -0,0 +1,7 @@
+---
+title: Wave v1.16.2
+date: 2024-12-20
+tags: [wave]
+---
+
```

Comment:
it should be updated to reflect the release notes here https://github.com/seqeralabs/wave/releases/tag/v1.16.2